src/main/java/com/researchspace/api/v1/controller/DocumentsApiController.java

```diff
@@ -144,7 +144,7 @@ public ApiDocumentSearchResult getDocuments(
      * Converting search results to ApiSearchResult
      */
     ApiDocumentSearchResult searchResult = new ApiDocumentSearchResult();
-    List<ApiDocumentInfo> info = new ArrayList<ApiDocumentInfo>();
+    List<ApiDocumentInfo> info = new ArrayList<>();
```

Comment:
Do you think it would be more performing in this case if you create a `LinkedList` here?

Reply:
I didn't actually make any change here, only removed the pre-java 7 type declaration. This arraylist is also only used as a return from the api method so there won't be any performance benefit in using a `LinkedList` here.